CMake/kwiver-depends-ffmpeg.cmake

```diff
@@ -0,0 +1,10 @@
+# Optional find and confgure VXL dependency
+
+option( KWIVER_ENABLE_FFMPEG
+  "Enable FFMPEG dependent code and plugins (Arrows)"
+  ${fletch_ENABLED_FFMPEG}
+  )
+
+if( KWIVER_ENABLE_FFMPEG )
+  find_package( FFMPEG REQUIRED )
```

Comment:
Do we care about minimum version of FFmpeg at this point?

Reply:
This change could be done in a follow-up branch.